### PR TITLE
First pass at developer desktop wp lp

### PR DESCRIPTION
**Pull request by caldav**
## Done

- [New landing page](https://www-ubuntu-com-canonical-websites-pr-4425.run.demo.haus/engage/developer-desktop): developer desktop whitepaper at `/engage/developer-desktop`
- [x] Copy doc
- [x] Correct form id and returnURL

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure [the page](https://www-ubuntu-com-canonical-websites-pr-4425.run.demo.haus/engage/developer-desktop) matches the copy doc, and that the downloadable is what's expected.

### CODE REVIEW

**Comment by anasereijo** [comment]
@caldav looking nice 👍

File: templates/engage/developer-desktop.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{% extends "engage/base_engage.html" %}
+
+{% block meta_description %}What's the most popular Linux OS? This whitepaper examines six key reasons why the developer community turn to Ubuntu.{% endblock %}
+
+{% block title %}Six reasons why developers choose Ubuntu Desktop{% endblock %}
+
+{% block meta_copydoc %}FIXME{% endblock meta_copydoc %}
```
Comment (shivjoshi1996) [inline templates/engage/developer-desktop.html]: Copydoc needs to be added (or code needs to be removed)

File: templates/engage/developer-desktop.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <ul class="p-list">
+            <li class="p-list__item is-ticked">Why the Ubuntu Desktop is the ideal platform to move through from development to production, whether for use in the cloud, server or IoT devices.</li>
+            <li class="p-list__item is-ticked">The extensive support and knowledge base available from the Ubuntu community, the wider Linux ecosystem and Canonical’s Ubuntu Advantage programme for enterprises. </li>
+            <li class="p-list__item is-ticked">How compatibility with a wide range of certified hardware and software enables a smooth, frictionless development process.</li>
+        </ul>
+
+
+    </div>
+    <div class="col-5" id="the-form">
+    {% include "../shared/forms/_landing_page_default.html" with id="FIXME" returnURL="FIXME" %}
```
Comment (shivjoshi1996) [inline templates/engage/developer-desktop.html]: ID & returnURL needs to be updated

**Comment by caldav** [comment]
@shivjoshi1996 fixed!

**Pull request merged**